docs/community-ee/community-ee-release-process.md

```diff
@@ -2,7 +2,7 @@
 
 ## Release Cadence
 
-The community EEs are built and published on the day after `ansible-core` is released.
+Ansible community Execution Environments(both Base and Minimal) are released the same week as the Ansible Community Package is released. For example: When `Ansible` 11.5.0 (with `ansible-core` 2.18.5) is released, both `Ansible Execution Environment Minimal and Base`  2.18.5-1 will also be released on that same week. Read about this more at https://docs.ansible.com/ansible/latest/getting_started_ee/index.html.
```

Comment:
```suggestion
Ansible community Execution Environments (both Base and Minimal) are released the same week as the Ansible Community Package is released. For example: When `Ansible` 11.5.0 (with `ansible-core` 2.18.5) is released, both `Ansible Execution Environment Minimal and Base`  2.18.5-1 will also be released on that same week. Read about this more at https://docs.ansible.com/ansible/latest/getting_started_ee/index.html.
```
I'm not  sure about the backticks here, though. They only make sense for `ansible-core` IMO (since that's a PyPI package name).